packages/web-app-files/tests/components/BatchActions.spec.js

```diff
@@ -0,0 +1,250 @@
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import DesignSystem from 'owncloud-design-system'
+import Vuex from 'vuex'
+import getters from '../../src/store/getters'
+import stubs from '../../../../tests/unit/stubs'
+import BatchActions from '../../src/components/BatchActions.vue'
+
+const localVue = createLocalVue()
+localVue.use(DesignSystem)
+localVue.use(Vuex)
+
+const selectedFiles = ['lorem.txt', 'owncloud.png']
+const componentStubs = { ...stubs, translate: true }
+const elSelector = {
+  copyButton: '#copy-selected-btn',
+  moveButton: '#move-selected-btn',
+  deleteButton: '#delete-selected-btn',
+  restoreButton: '#restore-selected-btn'
+}
+
+describe('Batch Actions component', () => {
+  const mountOptions = {
+    localVue
+  }
+
+  describe('files page', () => {
+    const $route = {
+      name: 'files-personal'
+    }
+
+    describe('when no items are selected for batch action', () => {
```

Comment:
I'd' `describe` is not necessary if it wraps only one test step. You could then have something like `it('should not display action buttons if no items are selected')`